src/app/lib/work/articipate/articipate-data.js

```diff
@@ -0,0 +1,13 @@
+const articipateData = {
+    hero: {
+        title: 'Articipate',
+        tagline: 'A mobile app that uses community to stimulate and challenge visual artists',
+        occasion: 'app design for '},
```

Comment:
Maybe add a newline character before `}` for style consistency within this file.